### ISSUE TITLE
Question about distillation loss implementation of ICARL 

### ISSUE THREAD

**Issue by pp1016** [RaptorMai/online-continual-learning#11]
Hi,

I found that in your ICARL Implementation, there is no hyper-parameter "T" used to soften the output distribution for distillation, which is implemented in other work such as LWF and BIC where T=2 is commonly suggested. Could you provide some detail for how you implement the distillation loss term? Thanks!

**Comment by pp1016** [comment]
Also they have the hyper-parameter to tune the influence between cross-entropy term and distillation term

**Comment by RaptorMai** [comment]
Hi, thank you so much for your interest in our work.

We followed Algorithm3 in the iCaRL paper, and it does not use “T” to soften the distribution and the hyperparameter to weigh the CE or KD term. 


We also implemented LWF, which contains the hyperparameter "T", as you can see in line 10 in kd_manager.py.  https://github.com/RaptorMai/online-continual-learning/blob/6175ca034e56435acd82b8f17ff59f920f0bc45e/utils/kd_manager.py#L10

For LWF, we did not use an additional hyperparameter to weigh the KD term because, in the LWF paper, the author said "λo is a loss balance weight, set to 1 for most our experiments." 

Please let me know if I answer your questions. Thanks!
